### PR TITLE
Add RunnableSequence documentation

### PR DESCRIPTION
**Pull request by eyurtsev**
Add RunnableSequence documentation

### CODE REVIEW

File: libs/langchain/langchain/schema/runnable/base.py [hunk 1]
```diff
@@ -1084,16 +1084,94 @@ def configurable_alternatives(
...
+    RunnableSequence in order.
+
+    A RunnableSequence preserves the streaming properties of its components, so if all
+    components of the sequence implement a `transform` method -- which
+    is the method that implements the logic to map a streaming input to a streaming
+    output -- then the sequence will be able to stream input to output!
+
+    If any component of the sequence does not implement `transform` then the sequence
+    will have to wait for all the components input to be generated, then wait
+    for the component generate output, before it can stream this component's output.
```
Comment (baskaryan) [inline libs/langchain/langchain/schema/runnable/base.py:1111]: maybe clearer? can't tell: 

If any component of the sequence does not implement `transform` then the streaming will only begin after this component is run. If there are multiple blocking components, streaming begins after the last one.

File: libs/langchain/langchain/schema/runnable/base.py [hunk 2]
```diff
@@ -1084,16 +1084,94 @@ def configurable_alternatives(
...
+
+    Please note: RunnableLambdas do not support `transform` by default! So if
```
Comment (baskaryan) [inline libs/langchain/langchain/schema/runnable/base.py:1113]: worth adding note?

if you want to implement custom functionality and support streaming, you'll need to create a new class that inherits from Runnable and specifies transform logic.

**Review by baskaryan — APPROVED** [review]

File: libs/langchain/langchain/schema/runnable/base.py (hunk 1 above)
Comment (eyurtsev) [reply libs/langchain/langchain/schema/runnable/base.py:1111]: nice

File: libs/langchain/langchain/schema/runnable/base.py (hunk 2 above)
Comment (eyurtsev) [reply libs/langchain/langchain/schema/runnable/base.py:1113]: yeah

**Pull request merged**